maggy/ablation/ablationstudy.py

```diff
@@ -341,6 +360,7 @@ def exclude_groups(self, *args, prefix=None):
     def print_all(self):
         """
         Prints all single layers that are included in the current ablation study configuration.
+
         """
```

Comment:
Single line docstring can be on a single line
`"""Prints all single layers that are included in the current ablation study configuration."""`